temporian/core/data/event.py

```diff
@@ -16,6 +16,7 @@
 
 from typing import List
 
+import temporian
```

Comment:
It would be best for internal code not to use the top level user api import

Reply:
Haven't found a way to circumvent this yet.

Reply:
I see two solutions:

- [Simple] The select op file included in the same build rule as the abstract operator.
- [Scallable] Create a registration mechanism (like this one https://github.com/google/temporian/blob/main/temporian/core/operator_lib.py) for ops to register to special callings on op (like [] and [[]]) in this case.

Reply:
Let's keep this for a next pr.